tasks/page/processBOM.js

```diff
@@ -17,12 +17,26 @@ if (require.main !== module) {
     } else {
       bom = path.join(boardInfo.boardPath, '1-click-bom.tsv')
     }
-    const deps = [
+    let kicadPcbFile
+    if (
+      boardInfo.eda &&
+      boardInfo.eda.type === 'kicad' &&
+      boardInfo.eda.pcb != null
+    ) {
+      kicadPcbFile = path.join(boardInfo.repoPath, boardInfo.eda.pcb)
+    } else if (boardInfo.eda == null) {
+      const kicadPcbPattern = path.join(boardInfo.boardPath, '**/*.kicad_pcb')
+      kicadPcbFile = globule.find(kicadPcbPattern)[0]
+    }
```

Comment:
Maybe we should give up on the `eda` field :thinking:

Reply:
Maybe. Out of scope for this one though, I guess.